### PR TITLE
Adding stable to stable upgrade for 4.12

### PR DESCRIPTION
**Pull request by JianLi-RH**
This PR will add 4.12 stable to 4.12 stable upgrade for amd64.

/cc @jhou1 @liangxia @shellyyang1989 @pruan-rht @bbguimaraes

### CODE REVIEW

**Comment by JianLi-RH** [comment]
Build versions can be found in:
https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/pr-logs/pull/openshift_release/35205/rehearse-35205-periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-stable-4.12-upgrade-from-stable-4.12-aws-ipi-byo-iam-role-p2-f14/1611606660604760064/artifacts/aws-ipi-byo-iam-role-p2-f14/cucushift-upgrade-helper-service-build-info/artifacts/ocp-build-info

The content is: `4.12.0-rc.5 4.12.0-rc.6`

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-stable-4.12-upgrade-from-stable-4.12.yaml [hunk 1]
```diff
@@ -0,0 +1,470 @@
...
+      product: ocp
+      relative: 2
+      version_bounds:
+        lower: 4.12.0-0
+        upper: 4.13.0-0
+  target:
+    prerelease:
+      architecture: amd64
+      product: ocp
+      relative: 1
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-stable-4.12-upgrade-from-stable-4.12.yaml:56]: We should test `relative: 1` to `relative: 0` (relative: 0 is the default, so can be omit)

**Comment by liangxia** [comment]
BTW, we also need to add the tests for arm and multi, too

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-stable-4.12-upgrade-from-stable-4.12.yaml (hunk 1 above)
Comment (JianLi-RH) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-stable-4.12-upgrade-from-stable-4.12.yaml]: Thanks!

**Comment by JianLi-RH** [comment]
@liangxia Yes, got it. I will start another RP for different arches once this got merged.

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-stable-4.12-upgrade-from-stable-4.12-azure-ipi-fips-p2-f7

**Comment by shellyyang1989** [comment]
Thanks @JianLi-RH !

I'm worried about our infra quota. The upgrade to stable build tests upgrade w/o force while the upgrade to nightly build tests upgrade w/ force. It seems to be the only difference. So it's fine for me to not cover all the workflows to save quota.

cc @jianlinliu for comments

**Comment by jianlinliu** [comment]
> Thanks @JianLi-RH !
> 
> I'm worried about our infra quota. The upgrade to stable build tests upgrade w/o force while the upgrade to nightly build tests upgrade w/ force. It seems to be the only difference. So it's fine for me to not cover all the workflows to save quota.
> 
> cc @jianlinliu for comments

Now we are having stable to nightly, stable to stable, also multiple arch, versions, and eus, if we always cover all workflow, quota definitely is going to be a problem for us. This deserve a brainstorm to have  a better test strategy to make everyone on the same page. I already list it as a topic in the next unified ci meeting.

**Comment by JianLi-RH** [comment]
needs-rebase

**Comment by JianLi-RH** [comment]
close this to fix conflict issue.

**Pull request closed without merging**